device/src/lib.rs

```diff
@@ -54,6 +54,27 @@ pub trait Timings {
     fn get_rx_window_duration_ms(&self) -> u32;
 }
 
+pub trait BoardEirp {
+    /// Returns the antenna gain in dBi. You should take into account any board losses here.
+    fn get_antenna_gain(&self) -> u8;
```

Comment:
Instead of getter methods, can't we use associated constants?
https://doc.rust-lang.org/reference/items/associated-items.html#associated-constants

Reply:
> Also, antenna gain could theoretically be negative..
That's a good point - thanks!

> Instead of getter methods, can't we use associated constants?
Yes, I think that's a good point. I thought maybe getters would be nice because it means it could vary during runtime, but I realized I can't think of a practical use-case for it yet, so let's just start with constants.

Writing the panic by hand seems a bit unnecessary though. I think the normal compiler error does a fine job?
```
35 |     const MAX_RADIO_POWER: u8;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^-
   |                              |
   |                              help: provide a definition for the constant: `= <expr>;
```